assets/css/_mixins.scss

```diff
@@ -29,3 +29,33 @@
     // There is no -moz- version
           touch-callout: default;
 }
+
+// Shared style mixins
+
+// Position and layout for error, loading, and blocking shield
+@mixin blocking-shield {
+  position: fixed;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  padding: 50px;
+  box-sizing: border-box;
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
```

Comment:
I'm far from expert in CSS/Sass but please just confirm that:
```sass

  padding: 50px;
  box-sizing: border-box;

  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
```
do the same as
```sass
  & > div {
    display: table-cell;
    text-align: center;
    vertical-align: middle;
  }
```
and
```sass
  & > div {
    display: table-cell;
    text-align: center;
    vertical-align: middle;
    padding: 0 50px;
  }
```